src/main/groovy/info/solidsoft/gradle/pitest/PitestPlugin.groovy

```diff
@@ -91,6 +92,13 @@ class PitestPlugin implements Plugin<Project> {
                 addPitDependencies(pitestConfiguration)
             }
         }
+        //for root project only
+        if (project.rootProject == project) {
+            project.tasks.register(PITEST_REPORT_AGGREGATE_TASK_NAME, AggregateReportTask) { t ->
+                t.description = "Aggregate PIT reports"
+                t.group = PITEST_TASK_GROUP
```

Comment:
`shouldRunAfter` after the `pitest` task?

Reply:
It seems that you forgot to fix it and that caused issues with functional tests on my machine. See below.